packages/react/src/components/Loader/Loader.tsx

```diff
@@ -102,18 +118,30 @@ class Loader extends UIComponent<ReactProps<LoaderProps>, LoaderState> {
   }
 
   renderComponent({ ElementType, classes, accessibility, variables, styles, unhandledProps }) {
-    const { indicator, label } = this.props
+    const { indicator, label, svg } = this.props
     const { visible } = this.state
 
+    const svgElement = Box.create(svg, {
```

Comment:
I am wondering whether the better shorthand for the svg would be the `Icon` component.. Can you elaborate why we are using the `Box` here?

Reply:
Make sense, I mention icon only because it is an svg